reoptjl/views.py

```diff
@@ -855,7 +855,10 @@ def queryset_for_summary(api_metas,summary_dict:dict):
     )
     if len(tariffOuts) > 0:
         for m in tariffOuts:
-            summary_dict[str(m.meta.run_uuid)]['year_one_savings_us_dollars'] = (m.year_one_energy_cost_before_tax_bau + m.year_one_demand_cost_before_tax_bau + m.year_one_fixed_cost_before_tax_bau + m.year_one_min_charge_adder_before_tax_bau) - (m.year_one_energy_cost_before_tax + m.year_one_demand_cost_before_tax + m.year_one_fixed_cost_before_tax + m.year_one_min_charge_adder_before_tax)
+            if len(tariffInputs) > 0:
+                summary_dict[str(m.meta.run_uuid)]['year_one_savings_us_dollars'] = (m.year_one_bill_before_tax_bau + m.year_one_export_benefit_before_tax_bau) - (m.year_one_bill_before_tax + m.year_one_export_benefit_before_tax)
```

Comment:
@Bill-Becker `m.year_one_export_benefit_before_tax` is actually a positive value if it's a benefit, so we should subtract this here if we want to account for it. However, I wonder if it makes more sense to just use `year_one_bill_before_tax` until we confirm how the value is used in the UI?

Reply:
Oops, and I agree - I updated